hyperspy/_signals/hologram_image.py

```diff
@@ -0,0 +1,303 @@
+            #  in the middle
+            yr, xc = np.unravel_index(eh_hw_fft_half.argmax(), eh_hw_fft_half.shape)  # Center of selected sideBand
+            sb_pos = [yr, xc]
+            # Default SB size is 1/2 of the distance to main band
+            sb_size = np.round(1.0 / 2 * (norm(np.subtract(sb_pos, [sx / 2, sy / 2]))))
+            # To be sure of even number, still questionable if it is needed:
+            sb_size = sb_size - np.mod(sb_size, 2)
+            rec_param = (xc - 1, yr - 1, xc + 1, yr + 1, sb_size)
+            print("Sideband Size in pixels")
+            print("%d" % sb_size)
+            #
+            # GUI based reconstruction, has to be redone using Hyperspy GUI:
+            # f, ax = plt.subplots(1, 1)
+            # ax.imshow(np.log(np.absolute(eh_hw_fft)),
+            #           cmap=cm.binary_r)  # Magnification might be added;
+            # # getting rectangular ROI
+            # rect = RoiRect()
+            # if hasattr(f.canvas.manager, 'window'):
+            #     f.canvas.manager.window.raise_()
+            # plt.waitforbuttonpress(100)
+            # plt.waitforbuttonpress(5)
+            # # Use this one in the case of usage of rect_roi obj:
+            # arect = eh_hw_fft[rect.y0:rect.y1, rect.x0:rect.x1]
+            # # Sideband position,find the max number and its [c,r]
+            # yR, xC = np.unravel_index(arect.argmax(), arect.shape)  # Center of selected sideBand
+            # # Use this one in the case of usage of rect_roi obj
+            # sb_pos = [np.round(rect.y0) + yR, np.round(rect.x0) + xC]
+            # a = 1.0
+            # sb_size = np.round(np.array([a / 3, a / 2, a]) *
+            #                    (norm(np.subtract(sb_pos, [sx / 2, sy / 2])) * np.sqrt(2)))
+            # # To be sure of even number, still questionable if it is needed:
+            # sb_size = sb_size - np.mod(sb_size, 2)
+            # print("Sideband range in pixels")
+            # print("%d %d %d" % (sb_size[0], sb_size[1], sb_size[2]))
+            # # Choose SideBand size
+            # sb_size = input("Choose Sideband size  pixel = ")
+            # sb_size = sb_size - np.mod(sb_size, 2)  # to be sure of even number...
+            # print("Sideband Size in pixels")
+            # print("%d" % sb_size)
+            # plt.close(f)
+            # rec_param = (rect.x0, rect.y0, rect.x1, rect.y1, sb_size)
+        else:
+            arect = eh_hw_fft[rec_param[1]:rec_param[3], rec_param[0]:rec_param[2]]
+            # Sideband position,find the max number and its [c,r]
+            yr, xc = np.unravel_index(arect.argmax(),
+                                      arect.shape)  # center of the selected sideBand
+            sb_pos = [np.round(rec_param[1]) + yr, np.round(
+                rec_param[0]) + xc]  # <----- use this one in the case of usage of rect_roi obj
+            sb_size = rec_param[4]
```

Comment:
Similar changes as above:
```python
yc, xc = np.unravel_index(arect.argmax(), arect.shape)  # center of the selected sideBand
sb_pos = (rec_param[1] + yc, rec_param[0] + xc)
sb_size = rec_param[4]
```
You should not need to round anything, because `rec_param` should just contain integers.